CodeQL migration: Java topics - change titles & add intros (2164) (#2866)
Semmle/ql: docs/language/learn-ql/java/source-locations.rst

Working with source locations
=============================

> You can use the location of entities within Java code to look for potential errors. Locations allow you to deduce the presence, or absence, of white space which, in some cases, may indicate a problem. 

Reviewer: The second sentence could perhaps be simplified?

```suggestion
You can use the location of entities within Java code to look for potential errors. Locations allow you to deduce how white space is used which, in some cases, may indicate a problem. 
```

Author: I think the longer version is clearer.
